backend/api/viewsets/Organization.py

```diff
@@ -36,6 +36,14 @@ def get_serializer_class(self):
         else:
             return self.serializer_classes['default']
 
+    def list(self, request):
+        fuel_suppliers = Organization.objects.filter(
+            type=OrganizationType.objects.get(type="Part3FuelSupplier")) \
```

Comment:
Could you add a comment here to explain why filtering on Part3FuelSupplier is being done?

Reply:
Sure. Updated.